tensorflow/core/platform/cloud/retrying_file_system.cc

```diff
@@ -40,20 +42,22 @@ bool IsRetriable(Status status) {
 Status CallWithRetries(const std::function<Status()>& f) {
   int attempts = 0;
   while (true) {
+    int delay = std::min(initial_delay_seconds << attempts, kMaximumBackoffSeconds);
```

Comment:
1. Will 'constexpr' compile though? I thought 'constexpr' has to be computable at compile time. 
2. I think it's fine to move 'attempts++' to make it the last line in the loop (i.e it could be after sleep())
3. In that case its semantics changes from "the number of made attempts" to "the current attempt number", so I suggest renaming 'attempts' to 'attempt' to reflect that
4. Also, 'attempts >= kMaxAttempts" in the 'if' line must be replaced with "attempt > kMaxAttempt" to keep the total amount of allowed attempts the same.